bin/hdfcoinc/pycbc_fit_sngls_by_template

```diff
@@ -74,6 +75,11 @@ parser.add_argument("--veto-file", nargs='*', default=[], action='append',
 parser.add_argument("--veto-segment-name", nargs='*', default=[], action='append',
                     help="Name(s) of veto segments to apply. Optional, if not "
                     "given all segments for a given ifo will be used")
+parser.add_argument("--gating-veto-windows", nargs='+',
+                    action=MultiDetOptionAction,
+                    help="Time in seconds to be vetoed before and after each "
+                         "gate. Given as detector-values pairs, e.g. "
+                         "H1:-1,2.5 L1:-1,2.5 V1:0,0")
```

Comment:
This isn't quite clear - the time *isn't* before and after the gate, as the gate already has finite size from the way the autogate works, and you use the central time.

So the question is really, does 'before and after each gate' refer to the central time (which is what is used here) or the edges of the gate - which is how I would understand it

I would suggest 'before and after the central time of each gate', but that may be too verbose

Reply:
You are totally right, thanks for pointing at it!